server/src/main/java/org/apache/kafka/server/config/KRaftConfigs.java

```diff
@@ -79,8 +77,7 @@ public class KRaftConfigs {
 
     public static final String CONTROLLER_LISTENER_NAMES_CONFIG = "controller.listener.names";
     public static final String CONTROLLER_LISTENER_NAMES_DOC = "A comma-separated list of the names of the listeners used by the controller. This is required " +
-            "if running in KRaft mode. When communicating with the controller quorum, the broker will always use the first listener in this list.\n " +
-            "Note: The ZooKeeper-based controller should not set this configuration.";
+            "in KRaft mode. When communicating with the controller quorum, the broker will always use the first listener in this list.";
```

Comment:
If we only have kraft mode, why do we need to say "in kraft mode"?
This implies there are other modes and is very confusing.